python/features/plugin_examples/root_keys_duplicate.aac

```diff
@@ -0,0 +1,16 @@
+schema:
```

Comment:
It's not easily evident what is wrong here, I think it would be more easily understood if you used the same model example from the unit test.

Reply:
Also, this isn't failing for the reason you believe. It is not actually triggering the unique root keys constraint. Can check the definition file manually to be sure you are triggering the constraint you are trying to trigger.